### PR TITLE
docs: editorial pass on running-exporters.md

### PR DESCRIPTION
**Pull request by zbwright**
This looks like it may have been a WIP. I took a pass through it before I noticed. Apologies if it was premature.

### CODE REVIEW

**Comment by zbwright** [comment]
@brancz

File: Documentation/user-guides/running-exporters.md [hunk 1]
```diff
@@ -60,14 +67,21 @@ spec:
...
 ### Namespace "limits"/things to keep in mind
+
 See the ServiceMonitor Documentation:
-> While `ServiceMonitor`s must live in the same namespace as the `Prometheus`
-resource, discovered targets may come from any namespace. This is important to allow
-cross-namespace monitoring use cases, e.g. for meta-monitoring. Using the
-`namespaceSelector` of the `ServiceMonitorSpec`, one can restrict the
-namespaces the `Endpoints` objects are allowed to be discovered from.
+
+> While `ServiceMonitors must live in the same namespace as the Prometheus
```
Comment (brancz) [inline Documentation/user-guides/running-exporters.md]: missing closing "`"

**Comment by brancz** [comment]
this document was contributed by the community and I preferred having something rather than nothing at the time :slightly_smiling_face:

**Comment by brancz** [comment]
@zbwright ping?

**Pull request merged**